repaygithub/cactus
# CACTUS-631 :: StatusStarFilled Icon

Ticket: https://repayonline.atlassian.net/browse/CACTUS-631

### Testing
1. Pull down this branch and run `yarn icons cleanup && yarn icons build`
2. Run `yarn icons start`
3. Have a look at the `StatusStarFilled` icon and make sure it looks good.
